packaging.mk

```diff
@@ -162,7 +162,8 @@ $(ARCHIVE_PREFIX)-windows-x86_64 $(ARCHIVE_PREFIX)-SNAPSHOT-windows-x86_64: \
 
 $(ARCHIVE_PREFIX)-%:
 	@rm -fr $@ && mkdir -p $@
-	cp $(filter-out build/apm-server-%, $^) $@
+# the apm-server.yml requires 644 permissions, let's avoid the issues with umask
+	install -m 640 $(filter-out build/apm-server-%, $^) $@
```

Comment:
maybe `600` to mach:
- https://github.com/elastic/apm-server/blob/e8b7251db2a12b777deca4b925f845b7e9ed87d6/packaging/nfpm.yml#L24-L31

Reply:
0600 sgtm

Reply:
```suggestion
	install -m 600 $(filter-out build/apm-server-%, $^) $@
```